bindings/nodejs/tests/assertions.js

```diff
@@ -1,13 +1,5 @@
 const assert = require('assert')
 
-function assertMessage(message) {
```

Comment:
why did you remove this?

Reply:
Wasn't sure how useful it is, will bring it back only with parents

Reply:
it's a utility function for the unit tests